v20.1/cockroachcloud-upgrade-to-v20.1.md

```diff
@@ -0,0 +1,163 @@
+
+- The [`GRANT`](grant.html) and [`REVOKE`](revoke.html) statements now require that the requesting user already have the target privileges themselves. For example, `GRANT SELECT ON t TO foo` requires that the requesting user already have the `SELECT` privilege on `t`.
+
+### Let ongoing bulk operations finish
+
+Make sure there are no [bulk imports](import.html) or [schema changes](online-schema-changes.html) in progress. These are complex operations that can increase the potential for unexpected behavior during an upgrade.</span>
+
+To check for ongoing bulk operations, use [`SHOW JOBS`](https://www.cockroachlabs.com/docs/v20.1/show-jobs.html#show-schema-changes) or check the [**Jobs** page](admin-ui-jobs-page.html) in the Admin UI.
+
+{{site.data.alerts.callout_info}}
+Once your cluster is running v20.1, but before the upgrade has been finalized, any ongoing schema changes will stop making progress, but [`SHOW JOBS`](show-jobs.html) and the [**Jobs** page](admin-ui-jobs-page.html) in the Admin UI will show them as running until the upgrade has been finalized. During this time, it won't be possible to manipulate these schema changes via [`PAUSE JOB`](pause-job.html)/[`RESUME JOB`](resume-job.html)/[`CANCEL JOB`](cancel-job.html) statements. Once the upgrade has been finalized, these schema changes will run to completion.
+
+Note that this behavior is specific to upgrades from v19.2 to v20.1; it does not apply to other upgrades.
+{{site.data.alerts.end}}
+
+### Review temporary limitations
+
+Once your cluster is running v20.1, but before the upgrade has been finalized:
+
+- New [schema changes](online-schema-changes.html) will be blocked and return an error, with the exception of [`CREATE TABLE`](create-table.html) statements without foreign key references and no-op schema change statements that use `IF NOT EXISTS`. Update your application or tooling to prevent disallowed schema changes during this period. Once the upgrade has been finalized, new schema changes can resume.
+
+- [`GRANT`](grant.html) and [`REVOKE`](revoke.html) statements will be blocked and return an error. This is because privileges are stored with table metadata and, therefore, privilege changes are considered schema changes, from an internal perspective. Update your application or tooling to prevent privilege changes during this period. Once the upgrade has been finalized, changes to user privileges can resume.
+
+   - This limitation also means that you will not be able to add or delete SQL users, or change existing users' passwords, on the [**SQL Users**](cockroachcloud-connect-to-your-cluster.html#step-2-create-a-sql-user) tab of the CockroachCloud Console. Attempting to do so will result in an error.   
+
+Note that these limitations are specific to upgrades from v19.2 to v20.1; they do not apply to other upgrades.
+
+## Step 4. Start the upgrade
+
+Once you have completed your preparations, starting the upgrade process is simple:
```

Comment:
Suggestion: `[...] starting the upgrade process is simple:` > `[...] you can start the upgrade process:`

I think documentation should try to stay away from words like "simple", since it's subjective (even if the process _is_ simple). This could be considered a personal style preference though, so it's ok if you ignore this suggestion.

Reply:
I agree. Simplifying to "To start the upgrade process:`.